website/src/scripts/setupSolidityDocs.ts

```diff
@@ -0,0 +1,38 @@
+const fileS = require("fs");
```

Comment:
Let's document these scripts somewhere, either in the README or as comment sdirectly in these files, that explain we're pulling generated documentation from `solidity-docgen`/TypeDoc into the website.